pkgs/development/libraries/zeromq/4.x.nix

```diff
@@ -12,12 +13,39 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-epOEyHOswUGVwzz0FLxhow/zISmZHxsIgmpOV8C8bQM=";
   };
 
+  patches =
+    # There is a bug in the CMakeLists.txt file where it doesn't
+    # correctly link to OpenPGM even if enabled:
+    # https://github.com/zeromq/libzmq/pull/4546
+    #
+    # This patch will likely need to be removed next time the zeromq version
+    # is bumped.
+    lib.optional enableOpenPGM
+      (fetchpatch {
+        url = "https://github.com/zeromq/libzmq/commit/5381be6c71f2a34fea40a29c12e3f6f8dcf3bad2.patch";
+        hash = "sha256-Wnmu0tlV3nWsnYWuXn6RtzVLNeHHTutnlPKGmZqV+oc=";
+      });
```

Comment:
Linking to this through github for posterity: https://github.com/zeromq/libzmq/pull/4546